pinc/stages.inc

```diff
@@ -508,8 +508,8 @@ function get_reserve_length($project, $round)
         // so are Children's Book Week projects (in 2017)
         if ($project->special_code == 'ChBookWk') return 0;
 
-        // Otherwise, reserve for 7 days.
-        return 7;
+        // Otherwise, reserve for 3 days.
+        return 3;
```

Comment:
This looks good, but lvl's comment in the task actually recommended making the French exception the same as the English That is, replace the two current special day exceptions (for anniversary and children's book week) with just:
```
// Same treatment as the English
// All 'special day' projects are exempted.
if ($project->special_code != '') return 0;
```

Reply:
Yup, you're right!